configs/setups/awicm3/awicm3.yaml

```
@@ -145,6 +148,19 @@ oifs:
                 "master":
                         with_xios: True
                         namelist_dir: ${general.esm_namelist_dir}/oifs/${version}/awicm3/master
+                        wam: false
+                        executable: OpenIFS
+                        cloudfixer: 0
+                        usemip: 0
```

```suggestion
                        mip: False
```

If you choose to follow my suggestions above, this is what you would need here.